java/client/src/main/java/glide/api/RedisClient.java

```diff
@@ -31,6 +31,6 @@ public static CompletableFuture<RedisClient> CreateClient(RedisClientConfigurati
 
     @Override
     public CompletableFuture<Object> customCommand(String[] args) {
-        return commandManager.submitNewCommand(CustomCommand, args, this::handleObjectResponse);
+        return commandManager.submitNewCommand(CustomCommand, args, this::handleObjectOrNullResponse);
```

Comment:
Renamed function to be consistent with how sometimes we allow for null and sometimes not.